ui/components/app/signature-request/signature-request.component.js

```diff
@@ -225,6 +226,11 @@ export default class SignatureRequest extends PureComponent {
           />
         </div>
         <div className="signature-request-content">
+          {txData?.securityProviderResponse?.flagAsDangerous !== 0 && (
+            <SecurityProviderBannerMessage
+              securityProviderResponse={txData?.securityProviderResponse}
+            />
+          )}
```

Comment:
From changes in snaps I am in doubt error message is showing up by default.

This will be true `txData?.securityProviderResponse?.flagAsDangerous !== 0`  even if `txData?.securityProviderResponse?.flagAsDangerous` is undefined.

Reply:
Yeah, good catch! The `securityProviderResponse ` was added into `beforeEach` and it affected all the tests including the ones that check matching the snapshot. It is fixed now and you can check the changes.